### PR TITLE
[miscellaneous] fix: update requirements

### PR DESCRIPTION
**Pull request by Wayonb**
problem: requirements were outdated
solution: update requirements

### CODE REVIEW

File: account/utils/SymbolAggregateBuilder.py [hunk 1]
```diff
@@ -19,13 +20,14 @@ def add_embedded_transaction(self, properties):
...
             **properties,
-            'type': 'aggregateComplete',
+            'type': 'aggregate_complete_transaction',
             'signer_public_key': self.signer_key_pair.public_key,
             'transactions_hash': self._calculate_transactions_hash(),
             'transactions': self.embedded_transactions
         })
 
-        aggregate_transaction.fee = (aggregate_transaction.get_size() + len(self.cosignatory_key_pairs) * COSIGNATURE_SIZE) * fee_multiplier
+        aggregate_transaction.fee = Amount((aggregate_transaction.size + len(self.cosignatory_key_pairs) * COSIGNATURE_SIZE) *
```
Comment (Jaguar0625) [inline account/utils/SymbolAggregateBuilder.py]: probably would add local variable to improve readability

File: account/symbol_multisig_relink.py [hunk 2]
```diff
@@ -39,13 +40,16 @@ def _prepare_aggregate_transfer(self, key_pair_repository, transaction_dict):
                 })
 
         aggregate_transaction = aggregate_builder.build(transaction_dict['fee_multiplier'], {'deadline': deadline})
-        aggregate_transaction.transactions[0].mosaics[0] = (transaction_dict['fee_mosaic_id'], aggregate_transaction.fee)
+        currency_mosaic = aggregate_transaction.transactions[0].mosaics[0]
```
Comment (Jaguar0625) [inline account/symbol_multisig_relink.py]: why assignment here, you're overwriting all fields aren't you?

File: templates/nem.mainnet.yaml [hunk 3]
```diff
@@ -9,4 +9,4 @@ accounts:
   - { address: NA6U4U4Z5J3L5ZKSMZD5DKPWSVT5DASJNO4V5RAA, name: Trading1, roles: [trading] }
 
 nodes:
-  - host: 54.170.97.221
+  - host: xem43.allnodes.me
```
Comment (Jaguar0625) [inline templates/nem.mainnet.yaml:12]: would probably add multiple nodes for redundancy?

File: network/harvester.py [hunk 4]
```diff
@@ -155,12 +158,13 @@ def main():
...
     parser.add_argument('--output', help='output file', required=True)
     parser.add_argument('--thread-count', help='number of threads', type=int, default=16)
+    parser.add_argument('--mosaic-id', help='mosaic id', default=MAINNET_XYM_MOSAIC_ID)
     args = parser.parse_args()
 
     resources = load_resources(args.resources)
     blocks_per_day = 60 if 'nem' == resources.friendly_name else 120
     downloader = HarvesterDownloader(resources, int(args.days * 24 * blocks_per_day), args.nodes)
-    downloader.download(args.thread_count, args.output)
+    downloader.download(args.thread_count, args.output, args.mosaic_id)
```
Comment (Jaguar0625) [inline network/harvester.py:167]: for proper handling, you'll probably also need to specify alias mosaic id [E74B99BA41F4AFEE] too

File: account/symbol_multisig_relink.py (hunk 2 above)
Comment (Wayonb) [reply account/symbol_multisig_relink.py]: yea, not sure what I was thinking 😆

File: templates/nem.mainnet.yaml (hunk 3 above)
Comment (Wayonb) [reply templates/nem.mainnet.yaml:12]: yea, I was trying to think of how to pull available node at runtime but will add more nodes for now.

File: network/harvester.py (hunk 4 above)
Comment (Wayonb) [reply network/harvester.py:167]: Is this needed for Harvesting?  I saw the alias mosaic getting used for transfers but didnt change that.  I guess you want to update the pulling of transfers also?

Comment (Jaguar0625) [reply network/harvester.py:167]: ah, ok, you're right. the single one should be fine for just account lookup.

**Review by Jaguar0625 — APPROVED** [review]
👍

**Pull request merged**